edge-modules/api-proxy-module/docker/linux/aarch64/Dockerfile

```diff
@@ -5,13 +5,10 @@
 
 ARG DIR=.
 
-FROM arm64v8/alpine:3.12.0
+FROM nginx@sha256:3a85694add68a9e7298d13627007340d17e5da9dcf172c89fc2a111eee805ca7
```

Comment:
nginx doesn't have tag? why use hash?

Reply:
nginx@sha256:3a85694add68a9e7298d13627007340d17e5da9dcf172c89fc2a111eee805ca7: I did that because I coud not find a better way to pull that exact version: linux/arm64/v8/nginx:1.19.2. See https://github.com/moby/moby/issues/36552